### DISCUSSION TITLE
The Village Well

### DISCUSSION THREAD

**Discussion by stgiga** [stgiga/UnifontEX#8] [General]
This is for chill conversation. One good topic would be what you plan to do or have done with UnifontEX. Also, if you have any ideas for the Wiki, I'd love to hear them.

**Comment by SuperPauly** [comment]
Do you plan on making a Nerd Font version for pretty terminals and code?

**Reply by stgiga** [reply]
I mean, there ARE codepoints in the font that map to even Powerline Extra, and quite a lot of `Miscellaneous Technical` and `Miscellaneous Symbols and Pictographs` corresponds to Font Awesome. Furthermore I haven't figured out how to properly add new glyphs. Also at present I don't have many glyph slots left. I should run out either Unicode 18 or 19. UnifontEX2 without glyph limits could definitely justify PUA usage. 

This isn't a no. But it's something beyond my *current* abilities.

**Comment by stgiga** [comment]
Well, I did some testing and B3K WOFF3 is most certainly the best option given mobile devices exist.
